vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_unblinded.cc

```diff
@@ -131,176 +114,405 @@ std::string GenerateTokenPayload(
   return json;
 }
 
-void Unblinded::Start(const std::string& viewing_id) {
+bool GetStatisticalVotingWinner(
+    double dart,
+    const double amount,
+    std::vector<ledger::ContributionPublisher> list,
```

Comment:
we can use `const std::vector<ledger::ContributionPublisher>&` here as well

Reply:
done